.github/workflows/deploy.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
     - uses: actions/checkout@master
```

Comment:
I missed this one. Should this be `actions/checkout@v4` as well? Using master is not recommended.

Reply:
Yes, if you want to create another PR for that, that would be appreciated! The CI here has not been touched for a while. The one in the PyPSA repo has been updated recently if you want to check it out as well.